atomic_reactor/schemas/config.json

```diff
@@ -665,6 +665,10 @@
                   }
               },
               "required": ["denylist_url", "denylist_key"]
+          },
+          "excludelist_content": {
```

Comment:
Is there a specific reason why this is called "excludelist" rather than "denylist"? This serves practically the same purpose as `denylist_srpms`, maybe a better name would be something like `denylist_container_first` or just `denylist_sources`?

Reply:
also for RPMS we had to switch to fetching config via HTTP because that list is maintained by somebody else and needs more often updates which may lead to updated OSBS config very often

Reply:
changed to : denylist_sources

Reply:
I noticed that the plugin already has a `get_denylisted_sources` helper method, but it gets denylisted SRPMs, which will now become confusing. Could it be renamed to `get_denylisted_srpms`? Could the methods/variables that have "excludelist" in the name be renamed as well?

Reply:
ok renamed what made sense